code/modules/projectiles/projectile.dm

```diff
@@ -2,12 +2,16 @@
 	name = "projectile"
 	icon = 'icons/obj/projectiles.dmi'
 	icon_state = "bullet"
-	density = 0
+	density = 1

```

Comment:
NO. This create a whole bunch of issues, I made a huge refactor PR just to make projectile non dense.